addons/purchase/controllers/portal.py

```diff
@@ -119,7 +127,11 @@ def resize_to_48(b64source):
         return self._get_page_view_values(order, access_token, values, history, False, **kwargs)
 
     @http.route(['/my/rfq', '/my/rfq/page/<int:page>'], type='http', auth="user", website=True)
-    def portal_my_requests_for_quotation(self, page=1, date_begin=None, date_end=None, sortby=None, filterby=None, **kw):
+    def portal_my_requests_for_quotation(self, page=1, date_begin=None, date_end=None, sortby=None, filterby=None, search=None, search_in='name', **kw):
+        if search_in and search:
+            search_domain = self._rfq_get_search_domain(search_in, search)
+        else:
+            search_domain = None
```

Comment:
`search_domain = self._rfq_get_search_domain(search_in, search) if search_in and search else None`